app/src/main/java/nodomain/freeyourgadget/gadgetbridge/activities/DiscoveryActivity.java

```diff
@@ -80,7 +80,7 @@
     private ScanCallback newLeScanCallback = null;
 
     // Disabled for testing, it seems worse for a few people
-    private final boolean disableNewBLEScanning = true;
+    private final boolean disableNewBLEScanning = false;
```

Comment:
@cpfeiffer 
No this reverts a change of mine, people had problems with the new BLE scanning (which was only used on newer Android versions), I disabled it for all since I saw no reason.
But maybe I missed something

Reply:
Some people had issues finding the BLE devices i am implementing, so i enabled it since people using my fork mostly only use for those specific devices anyway...

Would it be a solution to implement a setting to enable the new scanning method?

Reply:
I think we would start scanning with the method that would most probably work. That method may even depend on the Android version and/or the device capabilities. And then try the less likely methods.

A user probably has no idea which method to use. 

An alternative would be to first ask what kind of device to scan for and then decide on the method, but that would make it even less comfortable for the user.

Reply:
I still think that simply leaving a checkbox somewhere to switch to the new method would be sufficient for now...